static/src/javascripts/boot.js

```diff
@@ -33,6 +35,11 @@ define([
     var guardian = window.guardian;
     var config = guardian.config;
 
+    // global variable for sharing modules across bundles
+    window.guardian.app = window.guardian.app || {
```

Comment:
it's nit picky, but i wonder if should just define these all in one place? i.e. wherever `window.guardian` is defined? not a blocker though